versioned_docs/version-2.0.0/quickstart/node-express-mysql.md

```diff
@@ -0,0 +1,293 @@
+
+  sudo mkdir -p /usr/local/bin && sudo mv /tmp/keploy /usr/local/bin && keploy
+  ```
+
+  If everything goes right, your screen should look a bit like this:
+
+     <img src="/docs/img/code-snippets/install-keploy-logs.png" alt="Test Case Generator" width="50%" />
+
+  Moving on...
+     <details>
+     <summary style={{ fontWeight: 'bold', fontSize: '1.17em', marginLeft: '0.5em' }}> Run App with <img src="/docs/img/os/docker.png" alt="Docker Container" width="3%" /> Docker </summary>
+
+  #### Add alias for Keploy:
+
+  ```bash
+  alias keploy='sudo docker run --pull always --name keploy-v2 -p 16789:16789 --privileged --pid=host -it -v "$(pwd)":/files -v /sys/fs/cgroup:/sys/fs/cgroup -v /sys/kernel/debug:/sys/kernel/debug -v /sys/fs/bpf:/sys/fs/bpf -v /var/run/docker.sock:/var/run/docker.sock -v '"$HOME"'/.keploy-config:/root/.keploy-config -v '"$HOME"'/.keploy:/root/.keploy --rm ghcr.io/keploy/keploy'
+  ```
+
+  ### Lights, Camera, Record! 🎥
+
+  Fire up the application and MySQL instance with Keploy. Keep an eye on the two key flags:
+  `-c`: Command to run the app (e.g., `docker compose up`).
+
+  `--containerName`: The container name in the `docker-compose.yml` for traffic interception.
+
+  ```sh
+  keploy record -c "docker compose up" --containerName "nodesqlapp"
+  ```
+
+  🔥 Challenge time! Generate some test cases. How? Just **make some API calls**. Postman, Hoppscotch or even curl - take your pick!
+
+  #### Let's generate the testcases.
+
+  Make API Calls using [Hoppscotch](https://hoppscotch.io), [Postman](https://postman.com) or cURL command. Keploy with capture those calls to generate the test-suites containing testcases and data mocks.
+
+  ````bash
+  curl -X POST http://localhost:3000/items \
+     -H "Content-Type: application/json" \
+     -d '{"name": "Sample Item", "description": "This is a sample description"
+     }'
+
+    ```
+  Here's a peek of what you get:
+
+  ```
+  Item added with ID!
+  ````
+
+  🎉 Woohoo! With a simple API call, you've crafted a test case with a mock! Dive into the Keploy directory and feast your eyes on the newly minted `test-1.yml` and `mocks.yml`
+
+  _Time to perform more API magic!_
+  Follow the breadcrumbs... or Make more API Calls
+
+  ```bash
+  curl http://localhost:3000/items
+  ```
+
+  Or simply wander over to your browser and visit `http://localhost:8000/items`.
+  Did you spot the new test and mock scrolls in your project library? Awesome! 👏
+
+  #### Run Tests
+
+  Time to put things to the test 🧪
+
+  ```bash
+  keploy test -c "docker compose up" --containerName "nodesqlapp" --delay 10
+  ```
+
+  > The `--delay` flag? Oh, that's just giving your app a little breather (in seconds) before the test cases come knocking.
+
+  Your results should be looking all _snazzy_, like this:
+  <img src="/docs/img/testrun-node-fail.png" alt="Sample Keploy Test Result Gin MongoDB" width="100%" style={{ borderRadius: '5px' }}/>
```

Comment:
MongoDb is present instead of MySQL

Reply:
image added is incorrect as well

Reply:
https://github.com/Sonichigo/samples-typescript/blob/main/js-mysql/keploy/test-set-0/tests/test-1.yaml

please update based on the actual output of the test results, `ts` is not present in this application.